marketplace_builder/pages/getting-started/setup/deploying-syncing-changes.liquid

```diff
@@ -11,9 +11,9 @@ This guide will help you deploy and sync changes you make in your code base to y
 
 In order to deploy and sync changes to your Instance, you will need an Instance set up on our Partner Portal, and the Marketplace Kit installed on your computer. You will also need to have the code base for your Instance organized into the required directory structure—a way to set it up is by using the Blank Marketplace Module.
 
-* [Setting Up a Site on Our Platform]()
-* [Installing the Marketplace Kit]()
-* [Creating the Required Directory Structure]()
+* [Setting Up a Site on Our Platform](get-started/setting-up-site)
```

Comment:
i think its safer to start all links with `/`

Reply:
Thank you! I'm updating them.